student/models.py

```diff
@@ -16,6 +16,20 @@ def get_institutions_with_enrolled_students(cls):
         return cls.objects.filter(students__isnull=False)
 
 
+class Major(models.TextChoices):
+    UNDECIDED = "U", "Undecided"
+    COMPUTER_SCIENCE = "CS", "Computer Science"
+    SOFTWARE_ENGINEER = "SWE", "Software Engineer"
```

Comment:
I think that software engineer is more known as "SE"

Reply:
@guyy13 please resolve this or comment if you disagree so we can finally merge this PR

Reply:
Also think SE would be a better choice:) @guyy13

Reply:
I disagree
https://acronyms.thefreedictionary.com/software+engineer